.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 env:
   PLUGIN_NAME: FinSearch
-  PLUGIN_MIN_REQUIREMENT: ^6.5
+  PLUGIN_MIN_REQUIREMENT: ^6.5.3
```

Comment:
We decided to drop the support for lower versions, because of the change of Shopware